templates/learn/get-started.hbs

```diff
@@ -20,7 +20,10 @@
     <h3>{{fluent "learn-install-rustup-header"}}</h3>
     <p>{{fluent "learn-install-rustup"}}</p>
     {{> components/tools/rustup }}
-    <br /><br />
+    <br />
+    <h3>{{fluent "learn-install-rustup-update-header"}}</h3>
+    <p>{{fluent "learn-install-rustup-update"}} <code>rustup update</code>.</p>
```

Comment:
There's no guarantee that other languages will have sentence structures allowing this. This should instead be

```hbs
{{#fluent "learn-install-rustup-update"}}{{#fluentparam "update-command"}}<code>rustup update</code>{{/fluentparam}}{{/fluent}}
```

and

```ftl
learn-install-rustup-update = Rust updates very frequently. If you have installed Rustup some time ago, chances are your Rust version is out of date. Get the latest version of Rust by running { $update-command }
```

Reply:
Ah, thanks for that comment, I didn't think about that!